.mockery.yaml

```diff
@@ -0,0 +1,15 @@
+quiet: False
+disable-version-string: True
+with-expecter: True
+mockname: "{{.InterfaceName}}"
+filename: "{{.MockName}}.go"
+outpkg: mocks
+dir: "{{.InterfaceDir}}/mocks"
+packages:
+  github.com/Dynatrace/dynatrace-operator/src/builder:
```

Comment:
Also, it's not clear why we add only 1 mock to the config, but not others for instance registry mocks.

Reply:
isn't that a followup to do the migration of all the mocks ?

Reply:
I'm okay of the current state; my problem is that now we have config, but we don't list all existing mocks based on mockery. For instance, registries/mocks - are based on mockery. Also `repo/mocks/*` based on mocker too AFAIK.